src/kili/services/asset_import/base.py

```diff
@@ -427,6 +428,15 @@ def filter_duplicate_external_ids(self, assets):
             raise ImportValidationError(
                 "No assets to import, all given external_ids already exist in the project"
             )
+        nb_filtered_assets = len(assets) - len(filtered_assets)
```

Comment:
Isn't it simpler to go with `len(assets_in_project)`?

Reply:
I think that I used a miunderstanding name, it is nb_duplicate_assets